frontend/src/scenes/persons/RelatedFeatureFlags.tsx

```diff
@@ -0,0 +1,198 @@
+                return (
+                    <div>
+                        {featureFlagMatchMapping[featureFlag.evaluation.reason] || '--'}
+                        {matchesSet && (
+                            <span className="simple-tag ml-2" style={{ background: 'var(--primary-highlight)' }}>
+                                Set {(featureFlag.evaluation.condition_index ?? 0) + 1}
+                            </span>
+                        )}
+                    </div>
+                )
+            },
+        },
+        {
+            title: 'Status',
+            dataIndex: 'active',
+            sorter: (a: RelatedFeatureFlag, b: RelatedFeatureFlag) => Number(a.active) - Number(b.active),
+            width: 100,
+            render: function RenderActive(_, featureFlag: RelatedFeatureFlag) {
+                return <span className="font-normal">{featureFlag.active ? 'Enabled' : 'Disabled'}</span>
+            },
+        },
+    ]
+    return (
+        <>
+            <div className="flex justify-between mb-4">
+                <LemonInput
+                    type="search"
+                    placeholder="Search for feature flags"
+                    onChange={setSearchTerm}
+                    value={searchTerm}
+                />
+                <div className="flex items-center gap-2">
+                    <span>
+                        <b>Type</b>
+                    </span>
+                    <LemonSelect
+                        options={[
+                            { label: 'All', value: 'all' },
+                            {
+                                label: FeatureFlagReleaseType.ReleaseToggle,
+                                value: FeatureFlagReleaseType.ReleaseToggle,
+                            },
+                            { label: FeatureFlagReleaseType.Variants, value: FeatureFlagReleaseType.Variants },
+                        ]}
+                        onChange={(type) => {
+                            if (type) {
+                                if (type === 'all') {
+                                    if (filters) {
+                                        const { type, ...restFilters } = filters
+                                        setFilters(restFilters, true)
+                                    }
+                                } else {
+                                    setFilters({ type })
+                                }
+                            }
+                        }}
+                        value="all"
+                        dropdownMaxContentWidth
+                    />
+                    <span>
+                        <b>Match Evaluation</b>
+                    </span>
+                    <LemonSelect
+                        options={[
+                            { label: 'All', value: 'all' },
+                            ...Object.values(FeatureFlagMatchReason).map((value) => ({
```

Comment:
mocks seem to suggest this should be `All`, `match`, or `no match`

Reply:
✅